src/SprykerEco/Zed/Payone/Communication/Controller/GatewayController.php

```diff
@@ -37,8 +38,9 @@ class GatewayController extends AbstractGatewayController
      *
      * @return \Generated\Shared\Transfer\PayoneTransactionStatusUpdateTransfer
      */
-    public function statusUpdateAction(PayoneTransactionStatusUpdateTransfer $transactionStatusUpdateTransfer)
-    {
+    public function statusUpdateAction(
+        PayoneTransactionStatusUpdateTransfer $transactionStatusUpdateTransfer
+    ): PayoneTransactionStatusUpdateTransfer {
```

Comment:
It's also part of public API, please remove added type hints.

Reply:
Fixed.

Reply:
Fixed here and for whole file.